pkg/gitparse/gitparse_test.go

```diff
@@ -553,118 +577,18 @@ func TestLineChecks(t *testing.T) {
 
 	for name, test := range tests {
 		for _, pass := range test.passes {
-			if !test.function(false, pass.latestState, pass.line) {
+			if !test.function(pass.latestState, pass.line) {
 				t.Errorf("%s: Parser did not recognize correct line. (%s)", name, string(pass.line))
 			}
 		}
 		for _, fail := range test.fails {
-			if test.function(false, fail.latestState, fail.line) {
+			if test.function(fail.latestState, fail.line) {
 				t.Errorf("%s: Parser did not recognize incorrect line. (%s)", name, string(fail.line))
 			}
 		}
 	}
 }
 
-type messageParsingTestCase struct {
-	data     string
-	expected string
-}
-
-func TestMessageParsing(t *testing.T) {
```

Comment:
This was removed as we do not use the commit message at any point later in the pipeline. If we need it in the future we can always add it back.

Reply:
Do we scan commit messages? If not, we should.

Reply:
We do not currently, @zricethezav also mentioned there were plans to, but nothing definitive that i'm aware of.

Reply:
It's quite simple to implement with the current design, so I think it's worth considering how it could be done with this new implementation.

Reply:
Yea, I don't think it should be too much more complicated than with the prior implementation. (at least i hope not 😅 )

Reply:
> It's quite simple to implement with the current design, so I think it's worth considering how it could be done with this new implementation.

Sounds like another issue and implementation PR. We should wait before this PR is merged before considering it.